### ISSUE TITLE
Toaster not being displayed in the app window

### ISSUE THREAD

**Issue by dhimant5** [niklashenning/pyqttoast#24]
So i am working on a PySide6 app, but i am unable to show the toaster in the application window. I have a demo code below and the screen recording for the issue i'm facing
[screenrecording.webm](https://github.com/user-attachments/assets/2d011e86-7035-4a3c-898e-aae8e08f2a88)

```
from PySide6.QtWidgets import QMainWindow, QPushButton, QApplication, QWidget
from pyqttoast import Toast, ToastPreset

class Window(QMainWindow):
    def __init__(self):
        super().__init__()

        self.setWindowTitle("Toast Example")
        self.widget = QWidget(self)
        self.setCentralWidget(self.widget)

        self.button = QPushButton(self.widget)
        self.button.setText('Show toast')
        self.button.clicked.connect(self.show_toast)
    
    # Shows a toast notification every time the button is clicked
    def show_toast(self):
        toast = Toast(self.widget)  # Set parent as the main window
        toast.setDuration(5000)  # Hide after 5 seconds
        toast.setTitle('Success!')
        toast.setText('Woo hoo!')
        toast.applyPreset(ToastPreset.SUCCESS)  # Apply style preset
        toast.show()

if __name__ == "__main__":
    import sys

    app = QApplication(sys.argv)
    main_window = Window()
    main_window.show()
    sys.exit(app.exec())
```

**Comment by mateusmengatto** [comment]
Hello dhimant5, I didn't understand your issue. From the video, it seems that the QToaster is working correctly. I don't understand what you're trying to achieve, as QToaster is only meant for pop-up notifications.

**Comment by JimMcHale** [comment]
I am using PyQt6 and am running into the same issue. However its something i'm doing wrong.
If I modify demo/pyqt6/window.py to add:
window.py:316:        Toast.setPositionRelativeToWidget(self)
Then when one presses custom widget, you see the toast in the app window... So it  does work.

Must be something about me passing in a subwindow and trying to get the toast to appear in one of the  QGraphicsView that are in that window..

**Comment by niklashenning** [comment]
@dhimant5 The code you provided is working correctly, as it is supposed to show a toast relative to the screen. If you want to show the toasts relative to the application window, you have to call the static method `Toast.setPositionRelativeToWidget(some_widget)` with the widget you want the toasts to be displayed relative to. Just setting `self.widget` as the parent of the toast isn't supposed to make it relative to the application window (and isn't technically needed anyway — you could instantiate it without a parent and it wouldn't be any different).

This is how the code could look like (just added one line at the end of the `__init__` method):
```
from PySide6.QtWidgets import QMainWindow, QPushButton, QApplication, QWidget
from pyqttoast import Toast, ToastPreset


class Window(QMainWindow):
    def __init__(self):
        super().__init__()

        self.setWindowTitle("Toast Example")
        self.widget = QWidget(self)
        self.setCentralWidget(self.widget)

        self.button = QPushButton(self.widget)
        self.button.setText('Show toast')
        self.button.clicked.connect(self.show_toast)

        Toast.setPositionRelativeToWidget(self)

    # Shows a toast notification every time the button is clicked
    def show_toast(self):
        toast = Toast(self.widget)  # Set parent as the main window
        toast.setDuration(5000)  # Hide after 5 seconds
        toast.setTitle('Success!')
        toast.setText('Woo hoo!')
        toast.applyPreset(ToastPreset.SUCCESS)  # Apply style preset
        toast.show()


if __name__ == "__main__":
    import sys

    app = QApplication(sys.argv)
    main_window = Window()
    main_window.show()
    sys.exit(app.exec())
```

In your example, you would technically call the `Toast.setPositionRelativeToWidget()` method with `self.widget` instead of just `self`, but there is still a bug (#22) when using this method with a widget that is not a top level widget. I've been pretty busy so I haven't released a fix yet but it will be pushed soon with the next update :)

**Comment by JimMcHale** [comment]
Thank you! That fix for bug#22 fixed the issue I was having

**Comment by dhimant5** [comment]
@niklashenning sorry for the late reply, but this got it working
